pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -214,9 +214,19 @@ impl Lineage {
     ///
     /// Returns true if the Lsn was previously our branch point.
     pub(crate) fn is_previous_ancestor_lsn(&self, lsn: Lsn) -> bool {
+        // FIXME: this is insufficient even for path of two timelines for future wal recovery
+        // purposes
```

Comment:
nit: would it make sense to have an issue with more context and an example? I for one, don't understand what the problem is just from this comment

Reply:
I was planning to make it for the 3rd PR, but yes, I can make the comment better.